lib/querystring.js

```diff
@@ -162,6 +162,8 @@ function stringifyPrimitive(v) {
     return v;
   if (typeof v === 'number' && NumberIsFinite(v))
     return '' + v;
+  if (typeof v === 'bigint')
+    return '' + v;
```

Comment:
I think we should add support in `querystring.parse()` at the same time - it seems impossible to parse it back when you serialize a bigint as a normal number?

Reply:
I think that'd be less straightforward of a change and is probably better suited for discussion in a separate PR.

Reply:
@joyeecheung do you mean converting the values of the parsed objects to `number/bigint/string/boolean` because this already returns `true`:
```js
BigInt(querystring.parse(`a=${10n ** 1023n}`).a) === 10n ** 1023n
```

Reply:
hmm, I see, it'll always be parsed as a number so it's up to the user to decide which numeric type this field should be converted to. In that case nvm.